modules/installation-special-config-storage.adoc

```diff
@@ -127,44 +170,100 @@ boot_device:
   luks: <3>
     tpm2: true <4>
     tang: <5>
-      - url: https://tang.example.com
-        thumbprint: eyJhbmc3SlRyMXpPenc3ajhEQ01tZVJiTi1oM...
-    threshold: 1 <6>
-  mirror: <7>
-    devices: <8>
+      - url: http://tang.example.com:7500 <6>
+        thumbprint: PLjNyRdGw03zlRoGjQYMahSZGu9 <7>
```

Comment:
@jianlinliu are you able to review this section and/or this PR?

Reply:
I can review the change for the issues mentioned in https://bugzilla.redhat.com/show_bug.cgi?id=1937763 and https://github.com/openshift/openshift-docs/issues/33682, sound like they are fixed.

But the PR seem like making many other changes that I have no knowledge about. After go through the preview, I think the original doc chapter is more clear. User can easily follow the steps to select any type of disk encryption using machineconfig, and optional determinate if enable mirror disk, but now it sounds like combine everything together, also removing machineconfig methods. I am not sure this is on purpose and any background there. From my view, the original chapter is perfect.

Reply:
@jianlinliu thank you for your comment. This PR does not combine the disk encryption and mirroring sections, that was done previously in [PR 33337](https://github.com/openshift/openshift-docs/pull/33337). This current PR (34317) resolves the issues that you mentioned as outlined in the GitHub issue and the Bugzilla and it also enhances the module overall.

Reply:
@jianlinliu Note that this PR is a followup to an earlier one, https://github.com/openshift/openshift-docs/pull/33337, which performed the storage refactor you're referring to.  The current PR mainly improves wording and fixes bugs.

The previous approach was technically flawed for several reasons.  It required the mirroring config to be preprocessed through Butane and then bundled into an Ignition pointer config, bypassing machine configs entirely, which is a delicate process.  It also required substantially different procedures for LUKS depending on whether mirroring was also used.  Butane really needs to see all of the storage configuration at once, so it can generate an Ignition config with the correct storage hierarchy.  Thus, to avoid misleading users into thinking that mirroring and LUKS configuration could go into separate configs, we opted to combine the instructions into one procedure and one machine config that handles either or both.

Reply:
Got it. I can confirm the exchange key thumbprint and 7500 port change in this PR.

@jinyunma can you help change the existing machineconfig Tang disk encryption procedure to the new Butane procedure in 4.8

Reply:
Thank you for confirming that.